rwiki/rwiki-tool/tool/src/webapp/styles/wikiStyle.css

```diff
@@ -526,46 +527,66 @@ MOOT
 	-webkit-border-top-right-radius: 10px;
 	-moz-border-radius: 10px 10px 0 0;
 	padding: .5em;
-	background: #ddd
+	background: var(--sakai-background-color-2)
 }
 .rwikicomenttext{
 	-webkit-border-bottom-right-radius: 10px;
 	-webkit-border-bottom-left-radius: 10px;
 	-moz-border-radius: 0 0 10px 10px;
 	padding: 0 .5em;
 	width: auto;
-	background: #ddd;
+	background: var(--sakai-background-color-2);
 	padding: 0  10px  10px  10px
 }
-.rwikicomenttext p.paragraph{ margin: 0; padding: 0 }
+.rwikicomenttext p.paragraph{
+	background-color: var(--sakai-background-color-2);
```

Comment:
Not sure why Codacy flags this line - my code editor is set to 1 tab = 2 spaces and all the other indentation seems fine.

Reply:
@ern @profmikegreene

Reply:
I believe there's a rule for adding a space before the `{` but I don't know if that's the issue. I wouldn't worry about Codacy in this instance.